### PR TITLE
Wind test case

### PR DESCRIPTION
**Pull request by PaulTalbot-INL**
--------
Pull Request Description
--------
##### What issue does this change request address?
#178 

##### What are the significant changes in functionality due to this change request?
Adds a wind + peaker study case

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/HERON/wiki/Code-Standards) for details.
- [ ] 4. Automated Tests should pass.
- [ ] 5. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large tes.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added, the the analytic documentation must be updated/added.
- [ ] 9. If any test used as a basis for documentation examples have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: tests/workshop/wind/summary.csv [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
+,,,,,,,,,,,,,
```
Comment (PaulTalbot-INL) [inline tests/workshop/wind/summary.csv:21]: This looks right in excel; no reason to save it as an XLSX file though imo

**Comment by PaulTalbot-INL** [comment]
This case is the one Aaron will use in the upcoming July demo.

File: doc/user_manual/Makefile [hunk 2]
```diff
@@ -12,7 +12,7 @@ populate:
 	python script/generate_user_manual.py
 
 build: ../version.tex
-	cd build; pdflatex -interaction=nonstopmode $(SRCFILE).tex
+	cd build; pdflatex -interaction=nonstopmode $(SRCFILE).tex; pdflatex -interaction=nonstopmode $(SRCFILE).tex; pdflatex -interaction=nonstopmode $(SRCFILE).tex
```
Comment (dylanjm) [inline doc/user_manual/Makefile:15]: Do we have to build the docs 3 times in order to get the ToC correct?

File: doc/user_manual/src/HERON_user_manual.tex [hunk 3]
```diff
@@ -151,20 +151,22 @@
 \setcounter{tocdepth}{5} % include 5 levels of subsection in table of contents
 \title{HERON User Manual}
 \author{
-\textbf{\textit{Project Manager:}}
- \\Cristian Rabiti\\
  \textbf{\textit{Principal Investigator and Technical Leader:}}
 \\Paul W. Talbot\\
 \textbf{\textit{Main Developers:}}
 \\Paul W. Talbot\\
+Dylan J. McDowell\\
```
Comment (dylanjm) [inline doc/user_manual/src/HERON_user_manual.tex:158]: Thanks for adding me.

File: tests/workshop/wind/summary.csv (hunk 1 above)
Comment (dylanjm) [reply tests/workshop/wind/summary.csv:21]: Is this an analytic file or a file containing assumptions about the system?

**Review by dylanjm — APPROVED** [review]
These changes look good to me.

File: doc/user_manual/src/HERON_user_manual.tex (hunk 3 above)
Comment (PaulTalbot-INL) [reply doc/user_manual/src/HERON_user_manual.tex:158]: very deserved

File: doc/user_manual/Makefile (hunk 2 above)
Comment (PaulTalbot-INL) [reply doc/user_manual/Makefile:15]: yup, unless we get lucky, which apparently does sometimes happen. This seems to assure it happens.

**Comment by PaulTalbot-INL** [comment]
the `summary.csv` file has outputs from several perturbations of the workshop case, and is basically an analytic example, sure

**Pull request merged**